app/src/main/java/com/duckduckgo/app/browser/SwipingTabsFeature.kt

```diff
@@ -25,6 +25,15 @@ import com.duckduckgo.feature.toggles.api.Toggle
     featureName = "swipingTabs",
 )
 interface SwipingTabsFeature {
+    // The main kill switch for the feature
     @Toggle.DefaultValue(false)
     fun self(): Toggle
+
+    // The toggle used to enable the feature for internal users
+    @Toggle.DefaultValue(false)
```

Comment:
As mentioned on MM, you can just use `@InternalAlwaysEnabled`, so you don't need to have 2 separate subfeatures. You can have a single one with a rollout, so external users will get the feature enabled depending on rollout values, but internal ones will always get it